build/dockerfiles/Dockerfile

```diff
@@ -5,6 +5,7 @@ FROM ${BASE_DOCKER_IMAGE} as build
 RUN mkdir -p "$GOPATH/src/github.com/erda-project/erda/"
 COPY . "$GOPATH/src/github.com/erda-project/erda/"
 WORKDIR "$GOPATH/src/github.com/erda-project/erda/"
+ENV GOPROXY https://goproxy.cn/,direct
```

Comment:
Why add goproxy proxy ?

Reply:
Because when build image locally, it cannot download pkg sometimes without proxy.

Reply:
It may affect image build in hongkong, so removed it.